oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnectionRule.java

```diff
@@ -154,17 +203,45 @@ public ElasticConnection getElasticConnectionFromString() {
         }
     }
 
+    private boolean isValidUri(String connectionString) {
+        if (connectionString == null) {
+            return false;
+        }
+        try {
+            new URI(connectionString);
+            return true;
+        } catch (URISyntaxException e) {
+            return false;
```

Comment:
What about log.debug("Not a vlid URI: {}", connectionString, e) ? So that one can debug without having to run it in debug mode. As somebody might have a typo in the URI.

Reply:
Added log, exception itself have invalid url.